### PR TITLE
add variable for facetfilters

### PR DESCRIPTION
**Pull request by YiniXu9506**
# Pull request motivation(s)
Add variables to start_urls for facetfilters

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### What is the expected behaviour?
I want to use the version variable to filter the returned results

##### NB: Do you want to request a **feature** or report a **bug**?
Request a **feature**

##### NB2: Any other feedback / questions ?
No
@s-pace PTAL, thanks!

### CODE REVIEW

File: configs/pingcap.json [hunk 1]
```diff
@@ -4,50 +4,68 @@
     {
       "url": "https://pingcap.com/docs-cn/v3.0/",
       "selectors_key": "docs_cn",
+      "variables": {
```
Comment (s-pace) [inline configs/pingcap.json]: Variables are only used to dynamically inject values within the URL. Please use `extra_attribute` instead. You [can find an example here.](https://github.com/algolia/docsearch-configs/blob/master/configs/brew_all.json)

Comment (YiniXu9506) [reply configs/pingcap.json]: @s-pace Thanks!  Can I use the extra_attrobutes like below to filter results tag with cn and belong to v3.0?

```
index.search(
  {
    query: 'query string',
    facetFilters: ['tags: cn', 'version: v3.0'],
    hitsPerPage: 50,
  },
  (err, { hits } = {}) => {
    if (err) throw err;

    console.log(hits);
  }
);
```

Comment (s-pace) [reply configs/pingcap.json]: Yes indeed, please [mind the format of the facet filters](https://www.algolia.com/doc/api-reference/api-parameters/filters/#facet-filters). You should not include a space. 
```
    facetFilters: ['tags: cn', 'version:v3.0'],
```

Comment (YiniXu9506) [reply configs/pingcap.json]: Got it! Thanks! I have updated the extra_attributes, PTAL!

**Pull request merged**